pavelib/assets.py

```diff
@@ -64,6 +64,9 @@
     'cms': [path('lms') / 'static' / 'sass' / 'partials', ],
 }
 
+# Collectstatic log directory setting
+COLLECTSTATIC_LOG_DIR_ARG = "collect_log_dir"
```

Comment:
I understand that concern, but I thought my original feedback was that using `**kwargs` was preferred? The original code was considered to be too complicated and too far down the stack. Keeping it simpler and further up the stack implied using `kwargs`.

In part, the trouble is using a var to provide 3 different options: 
1. pass in "" for debug
2. pass in a specific directory
3. default to /dev/null

Is the thing we're talking about... a nit, a must, an ?   Should we pair on this instead?